doc/conf.py

```diff
@@ -109,7 +109,8 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 html_context = {
-    'css_files': ['_static/theme_overrides.css']
+    'css_files': ['_static/theme_overrides.css',
+                  '_static/collapsible_overrides.css']
```

Comment:
According to https://github.com/sphinx-doc/sphinx/issues/1379 you have to use the following code inside your extension instead of changing project config:
```python
env.app.add_stylesheet(stylesheet)
```

Reply:
I tried using this function in 'app' from `setup (app)` directly, and also in same 'env' we successfully use to add extension contents to '_static', but unlike adding file directly to 'html_context' it didn't work. Needs further investigation.

(Also '.add_stylesheet' was renamed into '.add_css_file')

Reply:
Move adding of theme_overrides.css into setup function in the conf.py, like below:
```
app.add_css_file('theme_overrides.css')
```
And do not set html_context at all.

Reply:
Also moved setting 'html_static_path' to `setup()` to allow it to be appended in extensions too.